ci/scripts/test-airgapped-build.sh

```diff
@@ -22,10 +22,11 @@ sudo ip netns exec airgapped ip link set dev lo up
 # Enter the network namespace and perform several builds.
 sudo ip netns exec airgapped sudo -u "$USER" bash -c \
   "export BAZEL_BITSTREAMS_CACHE=$(pwd)/bazel-airgapped/bitstreams-cache;
-  export BITSTREAM=--offline;
+  export BITSTREAM=\"--offline latest\";
   export BAZEL_PYTHON_WHEELS_REPO=$(pwd)/bazel-airgapped/ot_python_wheels;
-  bazel-airgapped/bazel build \
-    --distdir=bazel-airgapped/bazel-distdir \
-    --repository_cache=bazel-airgapped/bazel-cache \
-    //sw/device/tests:uart_smoketest_sim_dv"
+  bazel-airgapped/bazel build --sandbox_debug --verbose_failures \
```

Comment:
Probably want to delete `--sandbox_debug`, right?

Reply:
oops good catch!